python/nav/asyncdns.py

```diff
@@ -24,6 +24,8 @@
 
 """
 
+# ruff: noqa: F401
```

Comment:
As mentioned in another comment, unused imports were already disabled on a block-by-block basis for pylint (see W0611), not on a full-file basis.  Not sure if ruff can be disable for a specific block of code, or if it's always a choice between full-file and single lines.

Additionally, pylint was unable to detect that there is indeed such a thing as `twisted.internet.reactor`, which is why there is also a `E1101` disable before that.

I suspect the reason for the "unused" imports here is because these symbols are imported from here by other NAV modules that use the `asyncdns` module.

Reply:
Currently it is not possible to disable ruff for a specific block of code (https://github.com/astral-sh/ruff/discussions/3442#discussioncomment-6065662)